azure-app-service/massdriver.yaml

```diff
@@ -78,16 +78,40 @@ params:
               const: P3v3
         minimum_worker_count:
           title: Minimum instances
-          description: Minimum number of instances for autoscale.
+          description: Minimum number of instances for autoscale. (Enabling zone balancing multiplies this value by 3)
           type: integer
           minimum: 1
           maximum: 30
         maximum_worker_count:
           title: Maximum instances
-          description: Maximum number of instances for autoscale.
+          description: Maximum number of instances for autoscale.(Enabling zone balancing multiplies this value by 3)
           type: integer
           minimum: 1
           maximum: 30
+      dependencies:
+        location:
```

Comment:
I'd have to `mass app build` this to really know, but does this conflict with the `location` param above?

Reply:
Sweet, thank you